bin/ons_csv_to_ctb_json_read.py

```diff
@@ -50,6 +54,8 @@ def read(self):
                 raise ValueError(f'Reading {self.filename}: missing expected columns: '
                                  f'{", ".join(sorted(missing_columns))}')
 
+            dropped = 0
```

Comment:
Should this be called `dropped_by_filtering` to avoid confusion with the records that are dropped due to validation but not counted on `line 77`?

Reply:
Changed to `dropped_by_dataset_filter`